src/main/webapp/tutPopUp.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tutorial PopUp</title>
+    <link rel="stylesheet" href="style.css"></link>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
```

Comment:
We don't have a script.js file - all have more specific names.

Reply:
[We should use a lowercase name.](https://google.github.io/styleguide/jsguide.html#file-name)
~~~tutorial-populator.js

Reply:
i'm on that wave, we don't need any js for this file actually, I had just copied it from index.html, resolved now.